tests/core/commands/test_cli.py

```diff
@@ -320,7 +320,7 @@ def test_configuration_of_no_external_storage(isolated_runner, monkeypatch):
         monkey.setattr(StorageApiMixin, 'storage_installed', False)
         # Missing --no-external-storage flag.
         result = runner.invoke(cli, ['run', 'touch', 'output'])
-        assert 'is not configured' in result.output
+        assert 'External storage is not configured' in result.output
```

Comment:
configured or installed?

Reply:
Configured. Installed is checked somewhere else.